core/templates/dev/head/pages/exploration_editor/param_changes_editor_directive.html

```diff
@@ -47,9 +47,12 @@
       </span>
       <span>
         Change
-        <select2-dropdown item="paramChange.name" choices="paramNameChoices"
-                          placeholder="Param name" new-choice-regex="^[A-Za-z]+$"
-                          width="143px">
+        <select2-dropdown item="paramChange.name"
+                          choices="paramNameChoices"
+                          placeholder="Param name"
+                          new-choice-regex="^[A-Za-z]+$"
+                          width="143px"
+                          tag-mode="true">
```

Comment:
Ah I see, thanks for explaining.

Can we use a different name in our repo, for clarity? Maybe something like: can-add-new-choice? Or maybe use the presence/absence of new-choice-regex as the indicator for what is now "tag mode"?

Also, good catch on the TextInput thing. I think we don't support parameter evaluation when evaluating answers, so we probably don't have to worry about that bug for now.